src/bl/notificationFacade.ts

```diff
@@ -1,26 +1,26 @@
-import { getEntityManager } from 'typeorm'
+import { getManager } from 'typeorm'
 
 import { NotFoundException } from './errors/NotFoundException'
 import { Notification } from '../entities/notification'
 import { ParamsExtractor } from './paramsExtractor'
 
 export class NotificationFacade {
 
-    static async getAllFromUserId(userId: number): Promise<Notification[]> {
-        const notifications = await getEntityManager()
-                            .getRepository(Notification)
-                            .find({
-                                    user: userId
-                            })
-        if (notifications) {
-            return notifications
-        } else {
-            throw new NotFoundException('No Notification was found')
-        }
-    }
+    // static async getAllFromUserId(userId: number): Promise<Notification[]> {
+    //     const notifications = await getManager()
+    //                         .getRepository(Notification)
+    //                         .find({
+    //                                 user: userId
+    //                         })
+    //     if (notifications) {
+    //         return notifications
+    //     } else {
+    //         throw new NotFoundException('No Notification was found')
+    //     }
+    // }
 
     static async getById(notificationId: number): Promise<Notification> {
-        const notification = await getEntityManager()
+        const notification = await getManager()
```

Comment:
What is the difference, between entityManager and manager ?

Reply:
EntityManager is the old version of Manager in TypeORM.